requirements.txt

```diff
@@ -1,13 +1,12 @@
-Jinja2==2.7.2
-Markdown==2.4
-MarkupSafe==0.19
-Pygments==1.6
-Unidecode==0.04.14
-argparse==1.2.1
-blinker==1.3
-docutils==0.11
-feedgenerator==1.7
-pelican==3.1.1
-pytz==2014.2
-six==1.6.1
-wsgiref==0.1.2
+Jinja2>=2.7.2
+Markdown>=2.4
+MarkupSafe>=0.19
+Pygments>=1.6
+Unidecode>=0.04.14
+argparse>=1.2.1
+blinker>=1.3
+docutils>=0.11
+feedgenerator>=1.9
+pelican>=4.1
+pytz>=2014.2
+six>=1.6.1
```

Comment:
Is this the output of running ``pip freeze`` after the installation? If not, I'd prefer to do that since it locks the versions in.